src/features/reviews/components/AddReviewPrompt.vue

```diff
@@ -19,32 +19,46 @@ import { useRoute } from "vue-router";
 import MovieSearchPrompt from "../../../common/components/MovieSearchPrompt.vue";
 
 import { MovieSearchIndex } from "@/common/types/movie";
+import { useClubId } from "@/service/useClub";
+import { useDeleteListItem, useList } from "@/service/useList";
 import { useAddReview } from "@/service/useReview";
-import { useDeleteMovie, useWatchList } from "@/service/useWatchList";
 
 const emit = defineEmits<{
   (e: "close"): void;
 }>();
 
 const route = useRoute();
-
-const { data: watchList, isLoading: watchListLoading } = useWatchList(
-  route.params.clubId as string
+const clubId = useClubId();
+const { data: watchList, isLoading: watchListLoading } = useList(
+  clubId,
+  "watchlist"
```

Comment:
We should expose the enum types we created in the DB as the types of lists to the frontend so we don't need to pass these strings around

Reply:
Good point. I've updated all the `list` hooks to only accept `WorkListType` and updated all instances so we should get type errors now if we don't use the enum